docs/CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ To build some of the secDevLabs apps, some third party code, such as libraries o
 
 [1]:/docs/Makefile
 [3]:/docs/README_Template.md
-[4]:/docs/PR_Template.md
-[5]:/docs/Issue_Template.md
+[4]:https://github.com/globocom/secDevLabs/pulls
```

Comment:
Pull / Issue request template are now in [.github](https://github.com/globocom/secDevLabs/tree/master/.github) folder.

therefore, what do you think about just changing 46 line to:

```
Please let us know! Send a Pull Request with the `bug 🕷 ` label or create an Issue!
```

and remove those 56/57 lines (variables).